modules/core/src/test/scala/org/tessellation/http/routes/TrustRoutesSuite.scala

```diff
@@ -136,4 +141,34 @@ object TrustRoutesSuite extends HttpSuite {
     }
   }
 
+  test("GET previous peer-labels succeeds") {
+    val req = GET(uri"/trust/previous/peer-labels")
+
+    val gen = for {
+      trustMap <- genTrustMap
+      peerId <- peerIdGen
+      score <- Gen.chooseNum(0.1, 1.0).map(Refined.unsafeApply[Double, TrustValueRefinement])
+      seedlistEntries = Set(SeedlistEntry(peerId, none, none, score.some))
+      (_, publicTrust) <- genPeerPublicTrust
+      snapshotOrdinal = SnapshotOrdinal(1000L)
+      snapshotOrdinalPublicTrust = SnapshotOrdinalPublicTrust(snapshotOrdinal, publicTrust)
+    } yield (trustMap, seedlistEntries, peerId, snapshotOrdinal, snapshotOrdinalPublicTrust)
+
+    forall(gen) {
+      case (trust, seedlistEntries, peerId, snapshotOrdinal, snapshotOrdinalPublicTrust) =>
+        for {
+          ts <- mkTrustStorage(trust, seedlistEntries)
+
+          _ <- ts.updateNext(snapshotOrdinal)
+          _ <- ts.updateNext(peerId, snapshotOrdinalPublicTrust)
+          _ <- ts.updateCurrent(snapshotOrdinal.plus(500L))
+
+          routes <- mkPublicRoutes(ts)
+
+          expected <- ts.getBiasedSeedlistOrdinalPeerLabels.map(_.asJson)
+          testResult <- expectHttpBodyAndStatus(routes, req)(expected, Status.Ok)
+        } yield testResult
+    }
+  }
```

Comment:
I'm concerned about the hard-coded constants. Shouldn't be using the constants defined in `config` ?

Reply:
This is only a test. Does it matter?

Reply:
I believe it does because with the way it's written now one has to wonder if those values are the same as those in the config or if it's coincidence. Since the test relies on it matching those values it makes sense to pull the config out of your make function so you can access it and use them here.

Reply:
All of the magic numbers have been replaced by their `config` counterparts.